shared/teams/team/container.tsx

```diff
@@ -75,17 +74,23 @@ const Connected = Container.compose(
 
 class TabsState extends React.PureComponent<TabsStateOwnProps, {selectedTab: Types.TabKey}> {
   static navigationOptions = (ownProps: TabsStateOwnProps) => ({
```

Comment:
With the flag off the mobile header is funky
<img width="401" alt="image" src="https://user-images.githubusercontent.com/11968340/73959254-170b3380-48d7-11ea-9462-a7dd3679793b.png">